heron/metricscachemgr/src/java/com/twitter/heron/metricscachemgr/MetricsCacheManagerHttpServer.java

```diff
@@ -95,28 +92,12 @@ public static void main(String[] args)
       throws ExecutionException, InterruptedException, IOException {
     if (args.length < 3) {
       System.out.println(
-          "Usage: java MetricsQuery <topology_name> <component_name> <metrics_name>");
-    } else {
-      System.out.println("topology: " + args[0] + "; component: " + args[1]);
-    }
-
-    Config config = Config.newBuilder()
-        .put(Key.STATEMGR_ROOT_PATH,
-            System.getProperty("user.home") + "/.herondata/repository/state/local")
-        .build();
-    LocalFileSystemStateManager stateManager = new LocalFileSystemStateManager();
-    stateManager.initialize(config);
-
-    TopologyMaster.MetricsCacheLocation location =
-        stateManager.getMetricsCacheLocation(null, args[0]).get();
-    if (location == null || !location.isInitialized()) {
-      System.out.println("MetricsCacheMgr is not ready");
-      return;
+          "Usage: java MetricsQuery <host:port> <component_name> <metrics_name>");
     }
 
     // construct metric cache stat url
-    String url = "http://" + location.getHost() + ":" + location.getStatsPort()
-        + MetricsCacheManagerHttpServer.PATH_STATS;
+    String url = "http://" + args[0] + MetricsCacheManagerHttpServer.PATH_STATS;
+    System.out.println("endpoint: " + url + "; component: " + args[1]);
```

Comment:
is this `println` for debugging purpose？

Reply:
this is a tool.
it echo prints user input params.